src/main/java/Task/Duke.java

```diff
@@ -0,0 +1,123 @@
+        }
+        else if(userInput.startsWith("event")){
+            // create array to store words split by "/at"
+            String[] at = userInput.split("/at");
+            // take the description
+            String description = at[0].split("event")[1];
+            // create event task to be passed over to the actual task array
+            Event task = new Event(description,at[1]);
+            // assign task into actual task and increment listCounter
+            tasks[listCounter++] = task;
+        }
+        // print the newly added task
+        printAddedTask();
+    }
+
+    private static void printAddedTask() {
+        if (listCounter < 2) {
+            System.out.println("   Got it. I've added this task:\n\t"
+                    + tasks[listCounter-1] + "\n   Now you have "
+                    + listCounter + " task in the list.\n");
+        } else {
+            System.out.println("   Got it. I've added this task:\n\t"
+                    + tasks[listCounter-1] + "\n   Now you have "
+                    + listCounter + " tasks in the list.\n");
+        }
+    }
+
+    // view tasks
+    public static void getList(){
+        System.out.println("   Here are the tasks in your list:\n");
+        for(int i = 0; i < listCounter; i++){
+            System.out.println("   " + (i + 1) + ". " + tasks[i]);
+        }
+    }
+
+    private static void createLogo() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+    }
+
+    private static void greet() {
+        String greet = "Hello! I'm Duke\n"
+                + "What can I do for you?\n";
+        // print greet
+        System.out.println(greet);
+    }
+
+    private static void createByeMessage() {
+        String exit = "\nBye. Hope to see you again soon!\n";
+        System.out.println("   " + exit);
+    }
+
+    private static void setDone(String userInput) {
+        // take out the word after "done"
+        String indexString = userInput.split(" ")[1];
+        // change the word into integer
+        Integer indexTask = Integer.parseInt(indexString);
+        // mark task as done
+        tasks[indexTask-1].markAsDone();
+    }
+
+    public static void main(String[] args) {
+        // create logo
+        createLogo();
+        // greet
+        greet();
+        Scanner echo = new Scanner(System.in);
+        // scan user input
+        String userInput = echo.nextLine();
+
+        // execute command of the user input until "bye" is entered
+        while (!userInput.equals("Bye")){
+            // display list
+            if(userInput.equals("list")){
+                getList();
+            }
+            else if(userInput.startsWith("done")){
+                setDone(userInput);
```

Comment:
You may want to edit the format of the if-else statement such that it follows this form:
if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}
Just like what you did the the sections above.
Good Job!

Reply:
You may consider using switch cases. It will looks neater.

Reply:
Your while loop looks like missing a space. Maybe you can go to the coding standard and compare it .